script/bootstrap-old

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
```

Comment:
Instead of renaming the files to `bootstrap-old`, please remove the old files instead. This way the GitHub diff shows the difference between the files.